src/main/java/nextstep/ladder/domain/Result.java

```diff
@@ -0,0 +1,29 @@
+package nextstep.ladder.domain;
+
+import java.util.Objects;
+
+public class Result {
+    private final String value;
```

Comment:
단순히 `Result`와 `value`만으로는 어떤 결과인지 어떤 값을 가지고 있는 것인지 이해하기 힘들 수 있을 것 같아요 :)

Reply:
확실히 외부에서 봤을땐 인지하기 힘들 것 같네요!
```suggestion
public class LadderResult {
    private final String reward;
```
다음과 같은 네이밍정도로 바꿔보는게 어떨까 싶네요!